lib/theme/assets.dart

```diff
@@ -0,0 +1,3 @@
+abstract class Assets {
```

Comment:
@JElgar a helper class to not write `'assets/...png' inside view files, please let me know what do you think 🙌

Reply:
Clean thank you!